app/cloud_tasks/cloud_task_publishers.py

```diff
@@ -0,0 +1,57 @@
+from google import auth
+from google.cloud.tasks_v2 import CloudTasksClient, HttpMethod
+from google.cloud.tasks_v2.types.task import Task
+from structlog import get_logger
+
+from app.settings import SUBMISSION_CONFIRMATION_CONSUMER
+
+from .exceptions import CloudTaskCreationFailed
+
+logger = get_logger(__name__)
+
+
+class CloudTaskPublisher:
+    def __init__(self, queue_name: str):
```

Comment:
We should keep this similar to Pub/Sub where the queue name is provided when you call `create` as you could have multiple queues.